src/wolfgame.py

```diff
@@ -65,6 +65,7 @@
 var.LAST_TIME = None
 var.LAST_START = {}
 var.LAST_WAIT = {}
+var.LAST_GOAT = {}
 
 var.USERS = {}
 
```

Comment:
Just below this line, there is a `var.SPECIAL_ROLES` dict being created - you can remove that line.